x-pack/plugins/security_solution/public/common/links/links.test.ts

```diff
@@ -383,6 +386,9 @@ describe('security app link helpers', () => {
           id: 'hosts',
           path: '/hosts',
           title: 'Hosts',
+          landingImage: 'test-file-stub',
+          description:
+            'A computer or other device that communicates with other hosts on a network. Hosts on a network include clients and servers -- that send or receive data, services or applications.',
```

Comment:
nit: if most of this keys have the same value across different test cases would be nice to have a common structure to be used in all tests and modify what is needed for specifics.